var/spack/repos/builtin/packages/omm-bundle/package.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import shutil
+
+from spack.package import *
+
+class OmmBundle(MakefilePackage):
+    """Omm-bundle is a library implementing the orbital minimization method for 
+    solving the Kohn-Sham equation as a generalized eigenvalue problem and 
+    a bundle of four separate libraries: pspBLAS, MatrixSwitch, libOMM, tomato."""
+
+    homepage = "https://esl.cecam.org/"
+    git = "https://gitlab.com/ElectronicStructureLibrary/omm/omm-bundle.git"
+
+    version("master", branch="master")
+    version("1.0.0", tag="v1.0.0", commit="8b644267284695ff1a40b78d098bda6464a7b821")
+
+    depends_on("mpi")
+    depends_on("blas")
+    depends_on("lapack")
+    depends_on("scalapack")
+    depends_on("dbcsr")
+
+    patch("fjmpi_pspBasicTool.patch", when="@: %fj")
```

Comment:
Can you add a brief comment above the `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed
3. Where it was taken from

?

Reply:
@alalazo 
Comment added. This is a change to avoid an error that occurs when mpi.h is included more than once in the Fujitsu compiler.